src/tasks/runstart.ts

```diff
@@ -178,7 +179,17 @@ export const RunStartQuest: Quest = {
     {
       name: "Chateau Desk",
       completed: () => get("_chateauDeskHarvested") || !get("chateauAvailable"),
-      do: () => visitUrl("place.php?whichplace=chateau&action=chateau_desk"),
+      do: (): void => {
+        visitUrl("place.php?whichplace=chateau&action=chateau_desk");
+        const juice_bar_items: Item[] = [
```

Comment:
We use camelCase in TypeScript, so this should be:
```suggestion
        const juiceBarItems: Item[] = [
```